data/core/units/saurians/Javelineer.cfg

```diff
@@ -0,0 +1,98 @@
+#textdomain wesnoth-units
+[unit_type]
+    id=Saurian Javelineer
+    name= _ "female^Saurian Javelineer"
+    race=lizard
+    gender=female
+    image="units/saurians/javelineer/javelineer.png"
+    profile="portraits/saurians/skirmisher.webp"
+    hitpoints=50
+    movement_type=lizard
+    movement=7
+    experience=150
+    level=3
+    alignment=chaotic
+    advances_to=null
+    {AMLA_DEFAULT}
+    cost=47
+    usage=archer
+    description= _ "After years of combat practice, saurians warriors that are the most talented in the skill of spear throwing might advance to the ranks of the javelineers. With better equipment and exceptional aim, saurian javelineers are not to be underestimated."
```

Comment:
This definitely needs a different description at some point, but is not necessary for this PR.

Reply:
OK, I'll leave those as they are unless you want to propose something specific.